crates/biome_grit_patterns/tests/specs/ts/identifier.snap.new

```diff
@@ -0,0 +1,48 @@
+---
+source: crates/biome_grit_patterns/tests/spec_tests.rs
+assertion_line: 72
+expression: identifier
+---
+SnapshotResult {
+    messages: [],
+    matched_ranges: [
+        Range {
+            start: Position {
+                line: 1,
+                column: 4,
+            },
+            end: Position {
+                line: 1,
+                column: 4,
```

Comment:
The snapshot already highlights a bug, since the calculated line and column numbers are always the same, even if the bytes appear to be correct. I didn't want to start bug fixing in the same PR however, so this will be a follow-up fix :)